public/css/screen.styl

```diff
@@ -501,87 +501,336 @@ user-admin-border-size = 2px
 
 }
 
-#session-list {
+#topic-list {
   padding-top: 5px;
 }
 
-/*
- * Session display on event page
- */
+/* 
+  * Topic display on event page 
+*/
 
-.session {
+/* FF 3.5+ */
+body:not(:-moz-handler-blocked) .topic { 
+  width: 98%; 
```

Comment:
You might leave a more descriptive comment here for why this hack is needed.  I think I've guessed why (is it the scroll bar issue?) but it's not clear here.  Just "FF 3.5+" says to me that you're trying to make this selector only apply to FF 4 and so on... but I'm not sure if it's the other way around.